mcs/class/referencesource/System.Web/Configuration/ProcessModelSection.cs

```diff
@@ -60,6 +60,7 @@ namespace System.Web.Configuration {
             memoryLimit="60"
             webGarden="false"
             cpuMask="0xffffffff"
+            // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Not a secret.")]
```

Comment:
should be on the next line just before password ==

Reply:
Fixed by https://github.com/mono/mono/pull/20966